src/applications/find-forms/reducers/findVAFormsReducer.js

```diff
@@ -37,7 +37,8 @@ export default (state = initialState, action) => {
     case FETCH_FORMS_SUCCESS: {
       const clonedResults = cloneDeep(action.results);
       if (
-        !state.lighthouseTestSearchResults &&
+        (!state.lighthouseTestSearchResults ||
```

Comment:
I don't think I understand this expression, we're saying if it's `true || false`, which will always evaluate to `true` right?

Reply:
@kelsonic Yeah, this is a good catch, I just pushed up a refactor, I have it refactored for easier readability.